index.js

```diff
@@ -425,4 +429,18 @@
     return JSON.parse(s);
   });
 
+  //  regexp  /////////////////////////////////////////////////////////////////
```

Comment:
Vim's Replace mode comes in handy here. :)

For the benefit of Vim users who may read this, <kbd>Shift</kbd>-<kbd>R</kbd> switches to Replace mode.